pub-kube-config-files/upp-aggregate-healthcheck.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: upp-aggregate-healthcheck
+  labels:
+    app: upp-aggregate-healthcheck
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: upp-aggregate-healthcheck
+  template:
+    metadata:
+      labels:
+        app: upp-aggregate-healthcheck
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: aggregate-healthcheck
+        image: coco/upp-aggregate-healthcheck:v1.1.1-rc4
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 256Mi
+        env:
+        - name: PATH_PREFIX
```

Comment:
One more thing, since we will have 2 clusters now, please add ENVIRONMENT env var and I will also add it to delivery.
If we don't do this, agg-hc will take the default value of "Kubernetes" which will mess up things like grafana reports (all checks will be added to the same chart)